src/Microsoft.Sbom.Targets/GenerateSbomTask.cs

```diff
@@ -117,18 +119,12 @@ public GenerateSbomTask()
 
     public override bool Execute()
     {
-        // TODO: Validate all arguments.
-        if (string.IsNullOrEmpty(BuildDropPath) ||
-            string.IsNullOrEmpty(PackageSupplier) ||
-            string.IsNullOrEmpty(PackageName) ||
-            string.IsNullOrEmpty(PackageVersion))
-        {
-            Log.LogError("Required argument not provided.");
-            return false;
-        }
-
         try
         {
+            this.ValidateAndAssignVerbosity();
```

Comment:
Is this call necessary?

Reply:
Oh oopsies, I forgot to delete that. Thanks for catching it!

Reply:
Updated!